app/api/api_v1/schemas/metadata.py

```diff
@@ -6,6 +6,14 @@
 TaxonomyData = Mapping[str, Mapping[str, Union[bool, Sequence[str]]]]
 
 
+class OrganisationStats(BaseModel):
+    """Definition of stats used on homepage"""
+
+    total: int
+    laws: int
+    policies: int
```

Comment:
Could we also get the number of `UNFCCC` type?

Reply:
Done- tho I did rejig the structure to now include "organisations" so everything pertaining to an org is under there :smile: